### PR TITLE
[ZH] Update `Client/Keyboard_shortcuts`

### PR DESCRIPTION
**Pull request by QTPC**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by Pisapou** [comment]
Apply this for all your pr
- Don’t check the "changes are reviewed by a fluent speaker" until someone on GitHub reviewed this pr.

**Comment by QTPC** [comment]
> Apply this for all your pr
> 
> * Don’t check the "changes are reviewed by a fluent speaker" until someone on GitHub reviewed this pr.

ok

File: wiki/Client/Keyboard_shortcuts/zh.md [hunk 1]
```diff
@@ -155,137 +153,142 @@ tags:
...
+| `Ctrl` + `.` | 顺时针旋转90度。 |
+| `Ctrl` + `H` | 水平翻转。 |
+| `Ctrl` + `J` | 垂直翻转。 |
+| `Ctrl` + `G` | 反转，也就是将第一个物件与最后一个物件的位置交换，以此类推到所有已选物件上。 |
+| `Ctrl` + `Shift` + `R` | 旋转指定角度（弹出对话框）。 |
+| `Ctrl` + `Shift` + `S` | 缩放 （弹出对话框）。 |
+| `Ctrl` + `Shift` + `D` | 添加指定数量的多边形打击圈（弹出对话框）。 |
+| `Ctrl` + `Shift` + `F` | 把滑条转为连打（弹出对话框）。 |
+| `Ctrl` + `↑/↓/←/→` | 微调所选物件。（仅适用于 osu!，osu!taiko 和 osu!catch） |
+| `Ctrl` + `↑/↓/←/→` | 移动所选物件。（仅适用于 osu!mania） |
```
Comment (cdwcgt) [inline wiki/Client/Keyboard_shortcuts/zh.md:252]:
```suggestion
| `Ctrl` + `←/→` | 将所选物件向左或向右移动一列。（仅适用于 osu!mania） |
| `Ctrl` + `↑/↓` | 将所选物件向上或向下移动 1 个节拍细分。（仅适用于 osu!mania） |

```
1 个节拍细分不怎么好，可以考虑下改为更通顺的说法

File: wiki/Client/Keyboard_shortcuts/zh.md [hunk 2]
```diff
@@ -155,137 +153,142 @@ tags:
...
 
 | 快捷键 | 功能 |
 | :-- | :-- |
-| `T` | 按节奏轻按来校准歌曲的 BPM 和偏移量 |
-| 在调整 BPM 和偏移量时 `Shift` | 按住以四倍速调整 BPM 和偏移量 |
+| `T` | 按节奏轻按来校准歌曲的 BPM 和偏移量。 |
+| 在调整 BPM 和偏移量时 `Shift` | 按住以5倍正常的量调整。 |
+| 在调整BPM时 `Ctrl` | 按住以0.25调整。 |
+| 在调整偏移量时 `Ctrl` | 按住以1调整。 |
+| 在调整滑条速度时 `Ctrl` | 按住以0.1调整。 |
```
Comment (cdwcgt) [inline wiki/Client/Keyboard_shortcuts/zh.md:294]:
```suggestion
| 在调整 BPM 和偏移量时按住 `Shift` | 以 5 为单位调整。 |
| 在调整BPM时按住 `Ctrl` | 以 0.25 为单位。 |
| 在调整偏移量时按住 `Ctrl` | 以 1 为单位调整。 |
| 在调整滑条速度时按住 `Ctrl` | 以 0.1 为单位调整。 |
```
可能这段的英语也需要更改

**Review by cdwcgt — CHANGES_REQUESTED** [review]

**Comment by cdwcgt** [comment]
不用一直更新分支
也就是通过 `Update Branch` 合并 master 分支，不然会留下很多垃圾提交信息
只需要在准备好的时候让负责wiki的志愿者做就可以了

我晚上有空的时候会再看一遍

@QTPC

File: wiki/Client/Keyboard_shortcuts/zh.md [hunk 3]
```diff
@@ -12,92 +10,92 @@ tags:
...
+| `Shift` + `←/→` | 进入上一个/下一个分组。 |
+| `Shift` + `Enter` | 展开/折叠选择的分组。 |
+
+### 游戏模组
+
+*主页面：[游戏模组](/wiki/Game_modifier)*\
+*注：按下快捷键来打开或关闭 Mod。您必须进入 Mod 菜单才可以使用这些快捷键。*\
+*注：Target practice (osu!)，1K 和 2K (osu!mania)，Co-op (osu!mania)，还有 Mirror (osu!mania) 没有快捷键。*
 
 | 快捷键 | Mod |
```
Comment (cdwcgt) [inline wiki/Client/Keyboard_shortcuts/zh.md:100]:
```suggestion
| 快捷键 | 模组 |
```
漏了

**Review by cdwcgt — CHANGES_REQUESTED** [review]
除了这个就没问题了
I will approve after this fix

File: wiki/Client/Keyboard_shortcuts/zh.md (hunk 3 above)
Comment (QTPC) [reply wiki/Client/Keyboard_shortcuts/zh.md]: 修改了

**Review by cdwcgt — APPROVED** [review]

**Comment by cdwcgt** [comment]
内容是没问题了
但是提交记录可能需要rebase
剩下交给维护者就行了

The content of the article is ok, but it may need to be rebased

**Review by TicClick — APPROVED** [review]
it's alright -- the number of commits is relatively small, besides, we add a merge commit anyway prior to merging a branch into `master`, so it's not a big deal

**Pull request merged**